### PR TITLE
Adjust a few weird clang format updates

### PR DESCRIPTION
**Pull request by perrotta**
#### PR description:

See title
No changes expected

#### PR validation:

It compiles

### CODE REVIEW

**Comment by perrotta** [comment]
please test

**Comment by VinInn** [comment]
Le me understand:
each time this (and other files fixed by hand) will be edited and re-committed clang-format will run again and produce similar if not identical abominations...
How are we supposed to continue in a sustainable manner?

**Comment by perrotta** [comment]
> Le me understand:
> each time this (and other files fixed by hand) will be edited and re-committed clang-format will run again and produce similar if not identical abominations...
> How are we supposed to continue in a sustainable manner?

Well, the fixes here (that I only submitted "not to forget", since I took note of them, but I don't plan to pursue systematically) won't be reverted by a possible further run of clang tidy, since I moved the comment line somewhere else. I deliberately avoided fixing other similarly weird changes which risk to get reverted in the next run.

File: RecoLocalMuon/GEMSegment/plugins/ME0SegAlgoRU.cc [hunk 1]
```diff
@@ -380,12 +380,8 @@ bool ME0SegAlgoRU::areHitsCloseInEta(const float maxETA,
...
                                    << " and gp2 = " << h2 << " in eta part = " << h2.eta() << " ==> dEta = " << diff
                                    << " ==> return " << (diff < 0.1) << std::endl;
-  return (
-      diff <
-      std::max(
-          maxETA,
-          float(
-              0.01)));  //temp for floating point comparision...maxEta is the difference between partitions, so x1.5 to take into account non-circle geom.
+  return ( diff < std::max( maxETA, 0.01f));
+//temp for floating point comparision...maxEta is the difference between partitions, so x1.5 to take into account non-circle geom.
```
Comment (slava77) [inline RecoLocalMuon/GEMSegment/plugins/ME0SegAlgoRU.cc]: a comment aligned with the code and before the action is done seem to be more common.

BTW, does the text even make sense?
- ```temp for floating point comparision...``` sounds like it's justifying why ```float``` was added in the original code
- what is this "x1.5 ... " about?

Comment (perrotta) [reply RecoLocalMuon/GEMSegment/plugins/ME0SegAlgoRU.cc]: > BTW, does the text even make sense?
> 
> * `temp for floating point comparision...` sounds like it's justifying why `float` was added in the original code
> * what is this "x1.5 ... " about?

I wondered the same, but it was not my intention to inquire about it here.
As you know, even ```diff``` here makes no se sense (as it is a difference of ```h1.eta()``` with itself. But fixing that bug was not the scope of this PR

Comment (slava77) [reply RecoLocalMuon/GEMSegment/plugins/ME0SegAlgoRU.cc]: the code was modified (```float (0.01)``` -> ```0.01f``` ) and it apparently affects the content of the comment.

**Comment by makortel** [comment]
There is also a way to disable formatting for a region of code
```cpp
// clang-format off
...
// clang-format on
```
http://releases.llvm.org/3.6.0/tools/clang/docs/ClangFormatStyleOptions.html#disabling-formatting-on-a-piece-of-code

File: RecoLocalTracker/SiStripRecHitConverter/src/SiStripRecHitMatcher.cc [hunk 2]
```diff
@@ -96,8 +96,8 @@ void SiStripRecHitMatcher::match(const SiStripRecHit2D* monoRH,
   // position of the initial and final point of the strip in local coordinates (mono det)
   StripPosition stripmono = StripPosition(topol.localPosition(RPHIpointini), topol.localPosition(RPHIpointend));
 
-  if (trackdirection.mag2() <
-      FLT_MIN) {  // in case of no track hypothesis assume a track from the origin through the center of the strip
+// in case of no track hypothesis assume a track from the origin through the center of the strip
```
Comment (makortel) [inline RecoLocalTracker/SiStripRecHitConverter/src/SiStripRecHitMatcher.cc]: Shouldn't this comment be indented by two more spaces? Or did clang-format place it here?

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
- Technical
- Jenkins tests pass

**Comment by kpedro88** [comment]
+upgrade

**Comment by fabiocos** [comment]
+1

**Pull request merged**